plugins/modules/swim_intent.py

```diff
@@ -81,19 +81,20 @@
                 elements: dict
                 suboptions:
                   application_type:
-                    description: Swim Import Via Url's applicationType.
+                    description: Optional parameter indicating the type of application with permitted values(WLC, LINUX, FILREWALL, WINDOWS,
+                      LOADBALANCER, THIRDPARTY etc) applicable only for third party image types.
                     type: str
                   image_family:
-                    description: Swim Import Via Url's imageFamily.
+                    description: The name of image family applicable only in case of third party images upload (Optional).
                     type: str
                   source_url:
-                    description: Swim Import Image Via Url.
+                    description: Required parameter for importing swim image via remote url.
```

Comment:
A mandatory parameter for importing a SWIM image via a remote URL. This parameter is required when using a URL to import an image.

Reply:
added.